extensions-contrib/kafka-emitter/src/main/java/org/apache/druid/emitter/kafka/KafkaEmitterConfig.java

```diff
@@ -47,19 +74,76 @@
   @JsonCreator
   public KafkaEmitterConfig(
       @JsonProperty(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG) String bootstrapServers,
-      @JsonProperty("metric.topic") String metricTopic,
-      @JsonProperty("alert.topic") String alertTopic,
+      @Nullable @JsonProperty("event.types") Set<String> eventTypes,
```

Comment:
typically we let jackson de-serialize enum values directly, rather than doing the de-serialization in the code, so this would be defined as Set<EventType> and de-serialization would be defined as part of the EventType class.

This ensures serialization/de-serialization is handled consistently in all places where we might want to use that type, rather than having to redefine it in multiple places.

A good example is the JSONPathFieldType enum https://github.com/apache/druid/blob/84598fba3b283cbfd6a5addd2602c7b12ba8c00c/core/src/main/java/org/apache/druid/java/util/common/parsers/JSONPathFieldType.java#L26-L44

which can then get de-serialized directly to its final type in JSONPathFieldSpec https://github.com/apache/druid/blob/84598fba3b283cbfd6a5addd2602c7b12ba8c00c/core/src/main/java/org/apache/druid/java/util/common/parsers/JSONPathFieldSpec.java#L36